target/linux/ath79/image/generic.mk

```diff
@@ -694,6 +694,19 @@ define Device/librerouter_librerouter-v1
 endef
 TARGET_DEVICES += librerouter_librerouter-v1
 
+define Device/mikrotik_routerboard-wap-g-5hact2hnd
+  $(Device/mikrotik)
+  ATH := qca9556
+  DEVICE_MODEL := RouterBOARD wAP G-5HacT2HnD (wAP AC)
+  IMAGE_SIZE := 16000k
+  IMAGE/sysupgrade.bin := append-kernel | kernel2minor -s 1024 -e | \
+    pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | \
+    append-metadata | check-size $$$$(IMAGE_SIZE)
+  DEVICE_PACKAGES += kmod-ath10k-ct ath10k-firmware-qca988x-ct
```

Comment:
If this really is 64 MB RAM only, we should consider smallbuffers here (maybe @xback can even test that with/without).

Reply:
Done.